pcsx2/gui/Panels/SpeedhacksPanel.cpp

```diff
@@ -115,10 +115,10 @@ Panels::SpeedHacksPanel::SpeedHacksPanel( wxWindow* parent )
 	const wxSizerFlags sliderFlags( wxSizerFlags().Border( wxLEFT | wxRIGHT, 8 ).Expand() );
 
 	m_check_Enable = new pxCheckBox( this, _("Enable speedhacks"),
-		pxE( L"Speedhacks usually improve emulation speed, but can cause glitches, broken audio, and false FPS readings.  When having emulation problems, disable this panel first."
+		pxE( L"Speedhacks overall increases emulation speed, but can cause glitches, broken audio, and/or false FPS readings.\nWhen having emulation problems, disable (or alter) this panel first, in troubleshooting."
```

Comment:
*increase

Reply:
"overall increase emulation speed" isn't always true, hence the original wording.
Also I'd rather people disabled speedhacks when having problems, if you just alter them, you may notice no difference and then assume speedhacks aren't the problem.